### DISCUSSION TITLE
Passing 'mode' in JSON configuration

### DISCUSSION THREAD

**Discussion by elemdos** [tw-in-js/twind#88] [Q&A] [answered]
I'm wondering how to do this
```
import { setup, silent } from 'twind'

setup({
  mode: silent,
})
```
when passing my config in JSON?
```
    <script type="module" src="https://cdn.skypack.dev/twind/shim"></script>
    <script type="twind-config">
      {
        "hash": true
      }
    </script>
```
The docs mention that all the options can be passed here, but it seems like 'silent' is being stripped out because it's an object of methods. 

Thanks

**Comment by sastan** [comment, marked as answer]
> The docs mention that all the options can be passed here, but it seems like 'silent' is being stripped out because it's an object of methods.

From https://github.com/tw-in-js/twind/blob/main/docs/setup.md#mode

> If you are using JSON configuration the modes can be set using strings: "strict", "warn" or "silent".

```html
    <script type="module" src="https://cdn.skypack.dev/twind/shim"></script>
    <script type="twind-config">
      {
        "hash": true,
        "mode": "silent"
      }
    </script>
```

**Reply by elemdos** [reply]
Ah, sorry. Somehow missed that.
